rmgpy/data/solvation.py

```diff
@@ -277,28 +277,26 @@ def loadEntry(self,
                   shortDesc='',
                   longDesc='',
                   ):
-        spc = molecule
         if molecule is not None:
-            try:
-                spc = Species().fromSMILES(molecule)
-            except:
-                logging.debug("Solvent '{0}' does not have a valid SMILES '{1}'" .format(label, molecule))
-                try:
-                    spc = Species().fromAdjacencyList(molecule)
-                except:
-                    logging.error("Can't understand '{0}' in solute library '{1}'".format(molecule, self.name))
-                    raise
-            spc.generate_resonance_structures()
+            if not isinstance(molecule, list):
+                molecule = [molecule]
+            spc = []
```

Comment:
I would name this variable something like `spcs` or `spc_list` for consistency with expectations.